service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -96,9 +99,10 @@ class DatasetServiceTest {
           null);
 
   @BeforeEach
-  public void beforeEach() {
+  public void beforeEach() throws URISyntaxException {
```

Comment:
When writing tests, I think it's better to use `throws Exception` since we don't want to be updating our tests whenever a checked exception in an API is changed.

Reply:
This doesn't seem to be fixed yet.